digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -295,7 +308,13 @@ protected TaskResult callExecutor(Path projectPath, String type, TaskRequest mer
         DefaultSecretProvider secretProvider = new DefaultSecretProvider(
                 secretContext, secretAccessPolicy, grants, operatorSecretFilter, secretStore);
 
-        TaskExecutionContext taskExecutionContext = new DefaultTaskExecutionContext(secretProvider);
+        PrivilegedVariables privilegedVariables = GrantedPrivilegedVariables.build(
+                mergedRequest.getLocalConfig().getNestedOrGetEmpty("_env"),
```

Comment:
I assumed the purpose `_env` is for setting environment variables of the process spawned by `sh>` I.e. when the user intends to spawn a process. I.e., it only has meaning in the context of `sh>`.

Do you mean that `_env` would be used to control environment variables for e.g. processes spawned to execute a sandboxed non-sh operator? I.e. even when the user intent is not specifically to spawn a process?